### PR TITLE
Add JavaScript theme

### PR DESCRIPTION
**Pull request by Evavic44**
## Description

Add Javascript theme

Issue: #32

### Type of change

- [x] New feature (added a non-breaking change which adds functionality)
- [x] Updated documentation (updated the readme, templates, or other repo files)

## Checklist:

- [x] I have checked to make sure no other [pull requests](https://github.com/DenverCoder1/github-readme-streak-stats/pulls?q=is%3Apr+sort%3Aupdated-desc+) are open for this issue
- [x] The code is properly formatted and is consistent with the existing code style
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Javascript

### CODE REVIEW

**Comment by DenverCoder1** [comment]
Great work with the PR!

If you're willing to consider, maybe it could be improved with some design considerations.

* The white and the yellow have very low contrast with each other so it would be better to use a dark color for the flame (eg. black) to contrast with the light yellow background.

* The official ["JavaScript yellow"](https://github.com/colorjs/javascript-yellow) is `#f7df1e` so maybe consider that for the yellow parts.

* Since yellow can be difficult on the eyes, maybe consider swapping the colors (i.e. black background with yellow text and foreground shapes.) If you use a black background, you'd also be able to incorporate white in the foreground (for example for the streak and contribution numbers).

Let me know what you think.

**Comment by Evavic44** [comment]
Oh thank you, I thought that yellow was JavaScript color code. Will update it accordingly. 

You have some great points about the contrast, but I feel we should remain with the yellow, since the theme is titled JavaScript. 

What I suggest is to change the yellow background to the right JavaScript color code you shared _which is lighter to the eyes_ and use a dark color for the text and icon. 

What do you think? 
Background: `#f7df1e`
Text: `#000000`
Icon: `#000000`

If not, we can go with your idea. @DenverCoder1

**Comment by Evavic44** [comment]
Wow, they're both looking cool. Can I add them both? 😅

**Comment by DenverCoder1** [comment]
Sure!

**Comment by Evavic44** [comment]
I've updated the JavaScript theme.

image

File: docs/themes/README.md [hunk 1]
```diff
@@ -70,8 +70,12 @@ You can also try out and customize these themes on the [Demo Site](https://githu
 |       `deepBlue`       | ![image](https://user-images.githubusercontent.com/20955511/127893911-23e6c0c6-2d97-4028-a2b9-1fa1063db776.png) |
 |      `holi-theme`      | ![image](https://user-images.githubusercontent.com/62163030/129030429-0a4d48aa-d61f-4d0c-ae82-891d1b2476d7.png) |
 |       `ayu-light`      | ![image](https://user-images.githubusercontent.com/37243931/131808768-8617176a-fe84-4873-b9c2-f9ee74b30d87.png) |
+<<<<<<< HEAD
+|       `javascript`     | ![image](https://user-images.githubusercontent.com/62628408/135736923-ab029b34-baed-40cf-bca6-adcdfb319fd5.png) |
+=======
 |     `noctis-minimus`   | ![image](https://user-images.githubusercontent.com/20955511/135754306-58ceccde-90d9-46f9-a7b2-1cb54b6fbe24.png) |
 
+>>>>>>> upstream/main
```
Comment (DenverCoder1) [inline docs/themes/README.md:78]: Resolves merge conflict

```suggestion
|     `noctis-minimus`   | ![image](https://user-images.githubusercontent.com/20955511/135754306-58ceccde-90d9-46f9-a7b2-1cb54b6fbe24.png) |
|       `javascript`     | ![image](https://user-images.githubusercontent.com/62628408/135736923-ab029b34-baed-40cf-bca6-adcdfb319fd5.png) |
```

File: docs/themes/README.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ You can also try out and customize these themes on the [Demo Site](https://githu
 |      `holi-theme`      | ![image](https://user-images.githubusercontent.com/62163030/129030429-0a4d48aa-d61f-4d0c-ae82-891d1b2476d7.png) |
 |       `ayu-light`      | ![image](https://user-images.githubusercontent.com/37243931/131808768-8617176a-fe84-4873-b9c2-f9ee74b30d87.png) |
 |     `noctis-minimus`   | ![image](https://user-images.githubusercontent.com/20955511/135754306-58ceccde-90d9-46f9-a7b2-1cb54b6fbe24.png) |
-
+|       `javascript`     | ![image](https://user-images.githubusercontent.com/62628408/135736923-ab029b34-baed-40cf-bca6-adcdfb319fd5.png) |
```
Comment (DenverCoder1) [inline docs/themes/README.md:74]: Could you update the static png to have the new colors?

**Comment by Evavic44** [comment]
Added. 👍🏾 @DenverCoder1

**Review by DenverCoder1 — APPROVED** [review]

**Comment by DenverCoder1** [comment]
Thanks for contributing!

**Comment by Evavic44** [comment]
Thanks, @DenverCoder1

Adding some more themes now. 🙃

**Pull request merged**